src/code.py

```diff
@@ -69,9 +69,10 @@
     scale = 2
 
 launcher_config = {}
-if pathlib.Path("launcher.conf.json").exists():
-    with open("launcher.conf.json", "r") as f:
-        launcher_config = json.load(f)
+for launcher_config_path in ("launcher.conf.json", "/sd/launcher.conf.json"):
+    if pathlib.Path(launcher_config_path).exists():
+        with open(launcher_config_path, "r") as f:
+            launcher_config = launcher_config | json.load(f)
```

Comment:
would you be willing to refactor this into something like `launcher_config.py` in a function like `load_config()`? That way `boot.py` and the launcher, editor and any other apps that want to use it can do so without repeating the boilerplate. 

It might be nice if it would warn if it finds duplicated entries between the different files as well. I don't know how likely it is to come up, but it could be confusing if someone doesn't realize why some values get overwritten from another file.

I could also see value in having `/saves/launcher.conf.json` added to the list of files loaded perhaps since it's writable to circuitpython and can be modified in the editor without having to remount but doesn't require the extra hardware for an sdcard.